.pre-commit-config.yaml

```diff
@@ -56,6 +61,11 @@ repos:
         args: ["--recursive", "--settings-path", "./pyproject.toml", "."]
         language: system
         types: [python]
+        exclude: |
+          (?x)^(
+              .*/gunicorn_conf.py|
+              .*/test_gunicorn_conf.py
+          )$
```

Comment:
Why can't we run isort on those? 

And isn't it enough to mention them once on top of the file?

Reply:
https://github.com/pre-commit/pre-commit/issues/1966

When trying to run pre-commit on my local machine something like the above issue kept occurring. I had not modified the files but they kept appearing. Even when allowing them to go through pre-commit and get "fixed" then would fail the next run and then pass again. I'm not sure if it's a mac issue or pre-commit issue.

I had to put it in both locations because iSort ignored the top one.

Reply:
Resolving this for now; but I am going to make an issue to further diagnose this behavior